components/docs-chef-io/content/automate/major_upgrade_4.x.md

```diff
@@ -357,32 +486,95 @@ sudo ./chef-automate config show
 ```sh
 sudo ./chef-automate upgrade run --airgap-bundle automate-4.x.y.aib --major
 ```
+Here, you will be prompted to accept multiple Pre Upgrade checklist. Ensure you have perfomed all those actions before upgrade then mark yes, otherwise it will prompt you the error.The checklist will be as following:
 
-3. Check whether upgrade status is up-to-date
+```shell
+You had planned for a downtime, by running the command(chef-automate maintenance on)?: (y/n)
+y
+You have taken backup of your data and kept it safe, preferred on other disk or location? (y/n)
+y
+You are ready with your external open search with migrated data from elastic search, this should be done with the help of your Administrator (y/n)
+y
+After this upgrade completes, you will have to run Post upgrade steps to ensure your data is migrated and your Automate is ready for use (y/n)
+y
+
+Post Upgrade Steps:
+===================
+
+1) Check the status of your upgrade using:  
+     $ chef-automate upgrade status
+   This should return: Automate is up-to-date
+
+2) Check all services are running using: 
+     $ chef-automate status
+
+3) Disable the maintenance mode if you enabled previously using:
+	$ chef-automate maintenance off
+
+4) Check Automate UI everything is running and all data is visible
+
+**** In case of any errors, please refer to docs.chef.io and release notes for this version. ****
+
+Now, upgrade will start, Please confirm to continue... (y/n)
+y
 
-```sh
-sudo chef-automate status
 ```
 
-4. Upgrade your external *ElasticSearch 6.8* to *OpenSearch 1.2.4* manually. If you have configured *Host*, *Port*, *Username* or *Password* of ElasticSearch, patch the new configuration to use Chef Automate.
+It starts upgrading 
+Once you are done with the upgrade, follow the steps post upgrade which are:
 
-5. All [relevant configuration fields](https://docs.chef.io/automate/opensearch/) of the Elasticsearch should be copied into the OpenSearch configuration.
+```shell
+Post Upgrade Steps:
+===================
 
-    Please refer to the `elasticsearch.yml` file to get the applied configuration on your external Elasticsearch.
-    Add the releavent configuration from external Elasticsearch (`elasticsearch.yml`) to the `opensearch.yml` on your external OpenSearch.
+1) Check the status of your upgrade using:  
+     $ chef-automate upgrade status
+   This should return: Automate is up-to-date
+  
+2) Disable the maintenance mode if you enabled previously using:
+	$ chef-automate maintenance off
 
-6. Turn off maintenance mode using following command:
+3. Check whether upgrade status is up-to-date
```

Comment:
The formatting was not showing correctly on preview, for 3rd point
<img width="602" alt="image" src="https://user-images.githubusercontent.com/44021874/189196196-e7e18fa2-41b4-41aa-bc04-8478e35c0048.png">